frontend/src/slices/driver-table/actions.ts

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { toast } from 'react-toastify';
+
+import { HttpError } from '~/libs/packages/http/http.js';
+import { type AsyncThunkConfig } from '~/libs/types/async-thunk-config.type';
+import { type DriverGetAllResponseDto } from '~/libs/types/types.js';
+import { DriverActionMessage } from '~/packages/drivers/libs/enums/enums.js';
+import {
+  type DriverAddPayload,
+  type DriverCreateUpdateRequestDto,
+  type DriverGetDriversPagePayload,
+} from '~/packages/drivers/libs/types/types.js';
+
+import { ACTIONS_TYPES } from './common.js';
+
+const getDriversPage = createAsyncThunk<
+  DriverGetAllResponseDto,
+  DriverGetDriversPagePayload,
+  AsyncThunkConfig
+>(ACTIONS_TYPES.GET_DRIVERS_PAGE, async (payload, { extra }) => {
+  return await extra.driverApi.getPageOfDrivers(payload);
+});
+
+const addDriver = createAsyncThunk<
+  DriverCreateUpdateRequestDto,
+  DriverAddPayload,
+  AsyncThunkConfig
+>(ACTIONS_TYPES.ADD_DRIVER, async (payload, { rejectWithValue, extra }) => {
+  try {
+    const result = await extra.driverApi.addDriver(payload);
+    toast.success(DriverActionMessage.SUCCESS);
```

Comment:
In extra, we have our own custom notification service. You can use it like this: 
`extra.notification.success(DriverActionMessage.SUCCESS);`